docs/getting-started/workspaces.md

```diff
@@ -0,0 +1,94 @@
+   - **Description**: A brief summary of the workspace's purpose.
+
+<img width="1382" alt="Screenshot 2023-08-24 at 19 42 48" src="https://github.com/calimero-is-near/docs/assets/39309699/09f6d2d8-15db-4bc9-9d86-96f4b447e1b2"/>
+
+3. **Chain Details**: Add a unique identifier and description for your chain.
+
+<img width="1498" alt="Screenshot 2023-08-24 at 19 59 17" src="https://github.com/calimero-is-near/docs/assets/39309699/ab084db3-b97e-4ac8-8020-330d00442aa1"/>
+
+
+## Creating a Professional Mainnet Workspace
+
+For a professional workspace with mainnet deployment, follow these steps:
+
+1. **Select Workspace Type**: Choose the "Professional Workspace" option.
+
+2. **Mainnet Workspace**: Opt for the "Mainnet" environment if your smart contracts and applications are production-ready.
+
+<img width="1495" alt="Screenshot 2023-08-24 at 19 43 26" src="https://github.com/calimero-is-near/docs/assets/39309699/b80b0bfb-dc36-4339-b658-e6fc6ee74cb9"/>
+
+3. **Configuring Nodes**: Set up the number of validator nodes, RPC nodes, Archive RPC nodes, and Indexer nodes for your mainnet workspace.
+
+<img width="1332" alt="Screenshot 2023-08-24 at 19 46 43" src="https://github.com/calimero-is-near/docs/assets/39309699/6d7356ec-9695-4c92-ae24-5a5907d498da"/>
+
+
+4. **Chain Details**: Provide a unique identifier and description for your mainnet chain.
+
+<img width="1508" alt="Screenshot 2023-08-24 at 19 50 28" src="https://github.com/calimero-is-near/docs/assets/39309699/eadef6ef-0452-4026-afde-8d4ebd44098b"/>
+
+
+## Creating a Professional Testnet Workspace
+
+For a professional workspace with testnet deployment, follow these steps:
+
+1. **Select Workspace Type**: Choose the "Professional Workspace" option.
+
+2. **Testnet Workspace**: Opt for the "Testnet" environment if you are in the development, testing, or prototyping phase.
+
+<img width="1495" alt="Screenshot 2023-08-24 at 19 43 26" src="https://github.com/calimero-is-near/docs/assets/39309699/b80b0bfb-dc36-4339-b658-e6fc6ee74cb9"/>
+
+3. **Configuring Nodes**: Set up the number of validator nodes, RPC nodes, Archive RPC nodes, and Indexer nodes for your testnet workspace.
+
+<img width="1414" alt="Screenshot 2023-08-24 at 19 47 05" src="https://github.com/calimero-is-near/docs/assets/39309699/663d7f45-7c33-41aa-bfc8-ed023f71b217"/>
+
+4. **Chain Details**: Provide a unique identifier and description for your testnet chain.
+
+<img width="1508" alt="Screenshot 2023-08-24 at 19 50 28" src="https://github.com/calimero-is-near/docs/assets/39309699/eadef6ef-0452-4026-afde-8d4ebd44098b"/>
+
+Setting up workspaces within your organization allows you to tailor your blockchain environment to suit your specific needs.
+
+
+## Configure Testnet
+
+You'l need tto select the number of
+- validator nodes: They are running the consensus algorithm where threy calcuslate the value of treansations
+- RPC nodes: They allow you to communicare the directly with Calimero Shard
```

Comment:
Typo: `communicare`

I would change this to something like: "They allow you to directly query the state of the Calimero Shard chain"